lib/inputstreamhelper/__init__.py

```diff
@@ -540,6 +549,37 @@ def _chromeos_config(self):
 
         return devices
 
+    def _remove_old_backups(self, backup_path):
+        """Removes old Widevine backups, if number of allowed backups is exceeded"""
+        backups = int(get_setting('backups', '4'))
+        versions = sorted([LooseVersion(version) for version in os.listdir(backup_path)])
+
+        if 'x86' in self._arch():
+            installed_version = self._load_widevine_config()['version']
+        else:
+            installed_version = self._select_best_chromeos_image(self._load_widevine_config())['version']
+
+        while len(versions) > backups + 1:
```

Comment:
I think this can be more simple. It should return when `backups == 0` or when `len(versions) <= 1`.
And `remove_version = str(versions[1] if remove_version == installed_version else versions[0])`
So you can get rid of the exception-handling.

Reply:
So basically at the top:
```python
        if backups == 0 or len(versions) <= 1:
            return
```

And then here:
```python
        remove_version = str(versions[1] if remove_version == installed_version else versions[0])
```

Reply:
If the number of allowed backups is decreased, it has to remove multiple backups, so the while loop is necessary. I added a check at the top for `len(versions)` and simplyfied `remove_version`.

Reply:
Correct, that's an oversight of me.